lib/environment/hardhat.ts

```diff
@@ -22,7 +22,6 @@ export class NeokingdomDAOHardhat extends NeokingdomDAO {
     config: Partial<Config> = {}
   ) {
     const deployer = config.deployer ? config.deployer : await getWallet(hre);
-    const reserve = config.reserve ? config.reserve : deployer.address;
```

Comment:
should we throw when the reserve is not configured? just to avoid the deployer seeing their usdc mysteriously leaving their wallet...

Reply:
Good point